### PR TITLE
Beginning doc migration to GitHub

### PR DESCRIPTION
**Pull request by chapulina**
* Migrated most `bitbucket.org` links, except for 3 that reference PRs. Since we don't have PRs yet on GitHub I'm not sure what to link there.
* `mercurial` :arrow_right: `git`
* `default` :arrow_right: `master`

I haven't run through the tutorials, it would be ideal for whoever is reviewing to do so.

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -33,41 +33,43 @@ propose changes to this document in a pull request.
 ## Code of Conduct
 
 This project and everyone participating in it is governed by the [Ignition
-Robotics Code of Conduct](https://bitbucket.org/ignitionrobotics/ign-gazebo/src/default/CODE_OF_CONDUCT.md). By participating, you are expected to uphold this
+Robotics Code of Conduct](https://github.com/ignitionrobotics/ign-gazebo/src/master/CODE_OF_CONDUCT.md). By participating, you are expected to uphold this
```
Comment (maryaB-osr) [inline contributing.md:36]:
```suggestion
Robotics Code of Conduct](https://github.com/ignitionrobotics/ign-gazebo/blob/master/CODE_OF_CONDUCT.md). By participating, you are expected to uphold this
```

**Comment by maryaB-osr** [comment]
I can't leave a suggestion inline for this (I think because you didn't actually edit it in this branch), but the header link here, [contributing.md line 83](https://github.com/ignitionrobotics/docs/pull/8/files#diff-cc4aac3e9be04e0413c9520f223b493cR83) needs to be `(#repository-list)` not `(#markdown-header-repository-list)`

File: contributing.md [hunk 2]
```diff
@@ -132,21 +134,22 @@ existed.
 * **Check if you're using the latest software version**. A more recent version may contain your desired feature.
 * **Check if there's already [a library](https://ignitionrobotics.org/libs) which provides that enhancement.**
 * **Determine [which repository the enhancement should be suggested in](#markdown-header-repository-list).**
```
Comment (maryaB-osr) [inline contributing.md:136]:
```suggestion
* **Determine [which repository the enhancement should be suggested in](#repository-list).**
```

File: contributing.md [hunk 3]
```diff
@@ -132,21 +134,22 @@ existed.
...
 feature.** Someone else might already have started, and you might be able to
 help.
 
 #### How Do I Submit A (Good) Enhancement Suggestion?
 
-Enhancement suggestions are tracked as [Bitbucket
-issues](https://confluence.atlassian.com/bitbucket/issue-trackers-221449750.html).  After you've determined [which repository](#markdown-header-repository-list)
+Enhancement suggestions are tracked as [GitHub
+issues](https://help.github.com/en/github/managing-your-work-on-github/about-issues).
+After you've determined [which repository](#markdown-header-repository-list)
```
Comment (maryaB-osr) [inline contributing.md:146]:
```suggestion
After you've determined [which repository](#repository-list)
```

**Comment by maryaB-osr** [comment]
Lines 124 and 127 of `contributing.md` also need the `markdown-header` part of their links removed

**Comment by chapulina** [comment]
> (#repository-list) not (#markdown-header-repository-list)

I think that repo is private, but if you have access, [here](https://gitlab.com/ignitionrobotics/web/web/-/blob/master/src/app/doc/doc-version.component.ts#L48) you can see that the `markdown-header` is being stripped by the page generation code, like this:

```
if (href.startsWith('#markdown-header-')) {
        href = '#' + href.substring(17);
        // ...
}
```

I see all links within this page also have the `markdown-header` prefix.

**Comment by maryaB-osr** [comment]
Oops, ok then no other problems!

**Pull request merged**